### PR TITLE
Fixed travis evaluation of the acnv workflow

### LINKED ISSUE #680: Travis testing does not properly evaluates the acnv workflows
i.e. it will still pass even if one of the workflows fail

### PR DESCRIPTION
**Pull request by asmirnov239**
Closes #680 
Travis.yml wasn't properly evaluating the tests of the acnv workflows, as it did not fail if one of the workflows fails. 
Also I disabled QC for Travis tests since the CreatePanelOfNormals threw out all samples on the test data.

### CODE REVIEW

**Comment by asmirnov239** [comment]
@LeeTL1220 Could you please take a look?

**Comment by asmirnov239** [comment]
@LeeTL1220 Oooops, I need to fix the stuff with HetCoverage first

**Comment by asmirnov239** [comment]
@LeeTL1220 Finally done! I ended up setting stringency and readDepthThreshold low for WGS test, but kept default values for the WES test, as it was giving me a different type of error(I will send a detailed email). Could you please review it when you get a chance? Thanks!

File: scripts/wdl/README.md [hunk 1]
```diff
@@ -59,6 +60,8 @@ Reference used must be the same between PoN and case samples.
 - ``case_gatk_acnv_workflow.enable_gc_correction`` -- "true" is recommended. This is a true/false option change to false to bypass GC bias correction step.  *This must be the same as was used to generate the PoN*
 - ``case_gatk_acnv_workflow.isWGS`` -- Change to true if running WGS samples.  You should also change ``case_gatk_acnv_workflow.seg_param_undoSplits`` and ``case_gatk_acnv_workflow.seg_param_undoSD`` to reduce (not eliminate) hyper-segmentation.
 - ``case_gatk_acnv_workflow.wgsBinSize`` -- "10000" is recommended.  Size of bins (in bp) for WGS read counts CLI.  *This must be the same as was used to generate the PoN*.  Ignored if not running WGS.
+- ``case_gatk_acnv_workflow.stringency`` -- (Advanced) ``GetHetBayesianCoverage`` parameter, default=30
```
Comment (LeeTL1220) [inline scripts/wdl/README.md]: You should add a note that end users do not ever really have to change these values.

**Comment by LeeTL1220** [comment]
@asmirnov239 Minor comment.  Please address then feel free to merge.

File: scripts/wdl/README.md (hunk 1 above)
Comment (asmirnov239) [inline scripts/wdl/README.md]: done

**Pull request merged**